plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -65,6 +66,7 @@
     private DataSize targetSegmentPageSize = DataSize.of(1, MEGABYTE);
 
     @NotNull
+    @NotEmpty(message = "pinot.controller-urls cannot be empty")
```

Comment:
`NotEmpty` implies `NotNull`. Can remove the `NotNull` above. https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/javax/validation/constraints/NotEmpty.html